payments_service/braintree/views.py

```diff
@@ -129,22 +135,114 @@ def render(self, data, media_type=None, renderer_context=None):
+        log.debug('about to handle subscription: {s}'
+                  .format(s=result))
+
+        kind = result['braintree']['kind']
+        if kind !=  'subscription_charged_successfully':
+            log.warning('not notifying user of webhook kind: {k}'
+                        .format(k=kind))
+            return
+
+        # TODO: get the real product.
+        # https://github.com/mozilla/solitude/issues/481
+        product = products['mozilla-concrete-brick']
+
+        # TODO: use a real lookup in the config
+        # https://github.com/mozilla/payments-config/issues/8
+        sellers_by_product = {}
+        for seller_id, seller in sellers.items():
+            for p in seller.products:
+                sellers_by_product[p.id] = seller
+
+        bt_trans = result['mozilla']['transaction']['braintree']
+
+        # TODO: send an HTML email.
+        # https://github.com/mozilla/payments-service/issues/59
+        # TODO: get the actual seller name, not the ID.
+        # https://github.com/mozilla/payments-config/issues/7
+        # TODO: I think we need to get the actual price of the transaction here
+        # (localized).
+        # TODO: This assumes monthly subscriptions but we should figure that
+        # out automatically.
+        # TODO: link to terms and conditions for the payment.
+        body = textwrap.dedent("""
+
+        {seller.id}
+        {seller.url}
+
+        Date:                           {today_date}
+        Receipt #                       {transaction[uuid]}
+        Paid with                       {cc_type} ending in {cc_truncated_id}
+        {product.description}           {product.price[en]}
+        {bill_start} - {bill_end}       per month
+
+        TOTAL: {product.price[en]}
+        Next payment: {next_pay_date}
+
+        """).format(
+            product=product,
+            seller=sellers_by_product[product.id],
+            result=result,
+            today_date=datetime.today().strftime('%d %b %Y'),
+            transaction=result['mozilla']['transaction']['generic'],
+            # TODO: map type IDs to credit card names.
+            # https://github.com/mozilla/payments-config/issues/6
+            cc_type=result['mozilla']['paymethod']['type'],
+            cc_truncated_id=result['mozilla']['paymethod']['truncated_id'],
+            bill_start=parse_date(
+                bt_trans['billing_period_start_date']).strftime('%d %b %Y'),
+            bill_end=parse_date(
+                bt_trans['billing_period_end_date']).strftime('%d %b %Y'),
+            next_pay_date=parse_date(
+                bt_trans['next_billing_date']).strftime('%d %b %Y'),
+        )
+
+        # TODO: maybe localize the email?
+        # This will default to English.
+        send_mail("You're subscribed to {prod.description}"
+                  .format(prod=product),
+                  'Here is the message.',
+                  settings.SUBSCRIPTION_EMAIL_SENDER,
```

Comment:
Ideally set the from to noreply, and the reply-to to payments-support@mozilla.com so we don't get the bounces.